packages/main/src/themes/Calendar.css

```diff
@@ -15,7 +15,7 @@
 
 .ui5-cal-root [ui5-calendar-header] {
     height: var(--_ui5_calendar_header_height);
-    font-family: var(--_ui5_button_fontFamily);
+    font-family: var(--sapFontSemiboldDuplexFamily);
```

Comment:
For the calendar header buttons it should reflect the button's font-family I suppose.